editor/src/components/editor/store/editor-state.ts

```diff
@@ -1452,6 +1457,9 @@ export interface EditorState {
   githubSettings: ProjectGithubSettings
   imageDragSessionState: ImageDragSessionState
   githubOperations: Array<GithubOperation>
+  importOperations: Array<ImportOperation>
```

Comment:
I wonder if these should be under some umbrella field rather than at the root of `EditorState`.

Reply:
interesting point! which field, for example? I followed the `githubOperations` line, which also describe a temporary chain of operations, very similar to the `import` one